.github/workflows/benchmark.yml

```diff
@@ -0,0 +1,48 @@
+# Note: this workflow is currently being tested,
+# and so is only activated for the benchmark-test-cjs branch
+
+name: Performance benchmarks
+
+on:
+  push:
+    branches:
+      - benchmark-test-cjs
+
+jobs:
+
+  benchmarks:
+    runs-on: ${{ matrix.os }}
+
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest]
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python
+      uses: actions/setup-python@v1
+      with:
+        python-version: 3.8
+    - name: Install python dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install -e .[tests]
```

Comment:
I think we should install pinned dependencies using the `requirements-*.txt` files for the benchmark runs. Our CI builds should be as predictable as possible. We are explicitly installing from the unpinned dependencies in the `test-install` workflow to test whether there are any issues with this, everywhere else we should use the pinned dependencies.

Reply:
makes sense thanks, now changed